react-frontend/src/css/breadcrumbs.css

```diff
@@ -1,9 +1,10 @@
 .breadCrumbs {
-    margin-left: -10px;
-    padding-left: 5%;
     padding-top: 133px;
-    position: absolute;
-    width: 100%;
+    position: relative;
+    margin: auto;
+    margin-bottom: -133px;
+    width: 1045px;
```

Comment:
Wondering if this is part of the cause of #289 (causing the page to be too wide on smaller displays)?